canvas_account_admin_tools/static/canvas_account_admin_tools/js/services/ATRAPIService.js

```diff
@@ -0,0 +1,117 @@
+(function() {
+  var module = angular.module('ATRAPIModule', []);
```

Comment:
Nitpick:  I had mentioned this in the first pass :-). 
https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/pull/184/files/28b98b54157fc9ee590cb78a356b07a3b0c10f54#diff-f7d9736aa0774b6bc2ae795dc8809e20R2